src/Flex.php

```diff
@@ -298,7 +298,17 @@ public function update(Event $event, $operations = [])
             copy($rootDir.'/.env.dist', $rootDir.'/.env');
         }
 
-        $recipes = $this->fetchRecipes();
+        list($recipes, $conflicts) = $this->fetchRecipes();
+        if ($conflicts) {
+            $this->io->writeError(sprintf('<info>%d Conflict%s:</>', \count($conflicts), \count($conflicts) > 1 ? 's' : ''));
```

Comment:
I don't think any of the log messages are needed in this PR.
Which means all the logic can be moved inside fetchRecipes.
If we really want some log messages, it could be inside that method, in verbose mode only.

Reply:
Done, without warning, on verbose